internal/api/api.go

```diff
@@ -25,39 +26,58 @@ type snykApiClient struct {
 }
 
 func (a *snykApiClient) GetOrgIdFromSlug(slugName string) (string, error) {
-	url := a.url + "/v1/orgs"
-	res, err := a.client.Get(url)
+	u, err := OrgsApiURL(a.url, slugName)
 	if err != nil {
 		return "", err
 	}
+
+	res, err := a.client.Get(u.String())
+	if err != nil {
+		return "", err
+	}
+
+	//goland:noinspection GoUnhandledErrorResult
```

Comment:
Not everyone uses GoLand so I have mixed feelings about this. Could we standardize on golangci-lint as the single-source of truth for code quality & standards?

Reply:
Sure, wondering how to handle it without restricting our capability to leverage Golands quickfixes, though.